github/Organization.py

```diff
@@ -1156,6 +1157,33 @@ def get_migrations(self):
             headers={"Accept": Consts.mediaTypeMigrationPreview},
         )
 
+    def get_credential_authorizations(self):
+        """
+        :calls: `GET /orgs/:org/credential-authorizations <https://developer.github.com/v3/orgs>`_
+        :rtype: list
```

Comment:
list of :class:`github.AuthorizationOrganization.AuthorizationOrganization`_

(I'm not certain of the formatting, but roughly that, please)